trellis/trellis.go

```diff
@@ -103,21 +103,10 @@ func (t *Trellis) LoadProject() error {
 
 	os.Chdir(t.Path)
 
-	if os.Getenv("TRELLIS_VENV") != "false" {
+	if os.Getenv(TrellisVenvEnvName) != "false" {
 		if t.Virtualenv.Initialized() {
+			t.VenvInitialized = true
 			t.Virtualenv.Activate()
-		} else {
-			color.Yellow(`
```

Comment:
Refactored this because using anything other than `cli.Ui` for output will cause this output in _all_ tests which is super annoying and not good.

Now, each command just needs to do the check manually which might be better in the end anyway.